pkgs/misc/vim-plugins/vim-utils.nix

```diff
@@ -199,6 +199,10 @@ let
 
     linkVimlPlugin = plugin: packageName: dir: ''
       mkdir -p $out/pack/${packageName}/${dir}
+      if test -e "$out/pack/${packageName}/${dir}/${lib.getName plugin}"; then
+        echo -e "\nERROR - Duplicated vim plugin: ${lib.getName plugin}\n"
```

Comment:
Please use `printf` instead of `echo -e`. (printf is POSIX standard, echo with args is not.)

Reply:
Done, thanks @bjornfor